bakerydemo/static/css/main.css

```
@@ -14,81 +14,182 @@
 /* 9. Miscellaneous/ Helper styles ------------------------------------------ */
 
 /* Global styles */
+/* CSS Variables */
+:root {
+  --black: #000000;
+  --dark: #333333;
+  --grey: #6e6e6e;
+  --border-grey: #E1DCD3;
+  --white: #ffffff;
+  --cream: #f5f3e9;
+  --light-brown: #87744f;
+  --mid-brown: #825600;
+  --dark-brown: #553801;
+  --orange: #c55302;
+  --dark-orange: #833701;
+  --font--primary: "Marcellus", sans-serif;
+  --font--secondary: "Open Sans", sans-serif;
+}
+
+/* Utilities */
+
+.u-sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+/* Body */
+
 body {
+  font-family: var(--font--secondary);
+  font-size: 16px;
```

All font sizes should be defined in `rem` so the text can be resized with browser settings.

```suggestion
  font-size: 1rem;
```

---

I've refactored the typography across the site to follow this convention 👍